pages/contribute.md

```diff
@@ -2,7 +2,7 @@ We're really excited and grateful that you want to help with Exercism.
 
 There are three main ways to contribute to Exercism:
 - [Become a mentor](http://mentoring.exercism.io/) and help others learn a language.
-- Get involved in creating and managing a language track and eventually [become a maintainer](/become-a-maintainer).
```

Comment:
Hmm. This link points to the correct page - https://exercism.io/become-a-maintainer. I guess this change is not required?

Reply:
Or am I missing something? I am navigating from this page - https://exercism.io/contribute